### PR TITLE
i2c-tools: add license and tool

### PR DESCRIPTION
**Pull request by mans0n**
Maintainer: @dangowrt
Compile tested: ramips rt305x, RT5350F-OLinuXino, master
Run tested: ramips, RT5350F-OLinuXino, master

Description: 
- add libi2c license
- add i2ctransfer to tools

### CODE REVIEW

File: utils/i2c-tools/Makefile [hunk 1]
```diff
@@ -9,15 +9,15 @@ include $(TOPDIR)/rules.mk
...
+PKG_RELEASE:=2
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=@KERNEL/software/utils/i2c-tools
 PKG_HASH:=57b219efd183795bd545dd5a60d9eabbe9dcb6f8fb92bc7ba2122b87f98527d5
 
 PKG_MAINTAINER:=Daniel Golle <daniel@makrotopia.org>
-PKG_LICENSE:=GPLv2
-PKG_LICENSE_FILES:=COPYING
+PKG_LICENSE:=GPL-2.0-or-later LGPL-2.1-or-later
```
Comment (neheb) [inline utils/i2c-tools/Makefile]: Change the or later to a + as in other packages.

Comment (diizzyy) [reply utils/i2c-tools/Makefile]: Hmm? https://spdx.org/licenses/
It's currently using the correct SPDX "tags"

**Comment by karlp** [comment]
LGTM

File: utils/i2c-tools/Makefile (hunk 1 above)
Comment (neheb) [reply utils/i2c-tools/Makefile]: Hmm OK.

**Comment by diizzyy** [comment]
LGTM

**Pull request merged**